src/js/components/account/forgot-password.js

```diff
@@ -77,7 +77,7 @@ export class ForgotPassword {
           .empty()
           .append(
             "Something went wrong while communicating with the server. " +
-              "Please try again or contact support."
+              "Please try again or <a href='mailto:cape-agulhas-app@openup.org.za'>contact support</a>."
```

Comment:
I am wondering. We have this constants file: https://github.com/OpenUpSA/muni-portal-frontend/blob/master/src/js/components/constants.js

Perhaps we should add something like:

```javascript
export const SUPPORT_EMAIL = "cape-agulhas-app@openup.org.za";
```

Then we can do something like:

```javascript
import { SUPPORT_EMAIL } from ""../constants";

...

`"Please try again or <a href="mailto:${SUPPORT_EMAIL}">contact support</a>.`
```

Should we need to change the email address in the future, we then only need to do it in one place.

Reply:
Yeah I agree; I did think about doing the same, but we were already hard coding this in many places so I left it.

I think it is worthwhile; I'll do the refactoring in this PR. We'll just need to make sure we test it properly so nothing breaks.